src/Entity/Pap/Campaign.php

```diff
@@ -195,6 +195,46 @@ class Campaign implements IndexableEntityInterface, EntityScopeVisibilityInterfa
      */
     private $votePlaces;
 
+    /**
+     * @ORM\Column(name="delta_prediction_and_result_min_2017", type="float", nullable=true)
+     */
+    private ?float $deltaPredictionAndResultMin2017;
+
+    /**
+     * @ORM\Column(name="delta_prediction_and_result_max_2017", type="float", nullable=true)
+     */
+    private ?float $deltaPredictionAndResultMax2017;
```

Comment:
```suggestion
    private ?float $deltaPredictionAndResultMax2017 = null;
```

same for other fields